app/views/events/index.html.haml

```diff
@@ -5,13 +5,16 @@
   - events.each do |event|
     %div
       = link_to event.title, event_path(event)
-%p= link_to 'Create new event', new_event_path
+=link_to 'Create new event', new_event_path, :class => 'btn btn-flat create-event'
 %br
 %h3 List of all events
 %table.table.table-responsive
   - @events.each do |event|
     %tr
-      %td= link_to event.title, event_path(event)
-      %td= link_to 'Edit', edit_event_path(event)
-      %td= link_to 'Destroy', event_path(event), method: :delete, data:{confirm: 'Are you sure?'}
-%p= link_to 'Create new event', new_event_path
+      %td= link_to event.title, event_path(event), :class => 'link-underlined link-underlined-yellow'
```

Comment:
This is just a minor formatting comment: After "%td" I would start a new line like you did in the following lines.

Reply:
Nice catch. Changed it 🙂